src/MainWindow.vala

```diff
@@ -314,9 +301,9 @@ namespace PantheonCalculator {
 
             entry.grab_focus ();
 
-            show_all ();
+            present ();
 
-            this.key_press_event.connect (key_pressed);
+            // key_press_event.connect (key_pressed);
```

Comment:
Replacement for this is event controllers: https://valadoc.org/gtk4/Gtk.EventControllerKey.html

Reply:
The Problem that we are using key_press_event to change the keyval in the event, what, i believe, isn't possible with the event controllers.

however, should we be using [Gtk.Editable.insert_text](https://valadoc.org/gtk4/Gtk.Editable.insert_text.html) here?